src/Infrastructure/GoogleListingsAndAdsPlugin.php

```diff
@@ -56,6 +57,12 @@ public function activate(): void {
 		}
 
 		flush_rewrite_rules();
+
+		/** @var Activated $activated_event */
+		$activated_event = $this->container->get( Activated::class );
+		if ( ! empty( $activated_event ) ) {
```

Comment:
I realize that the use of `conditionally_share_with_tags` for registering these track event classes would require us to check for this but I don't think any of the event classes implement the `Conditional` interface. This means that they're always loaded. Maybe we could replace all those `conditionally_share_with_tags` with simple `share_with_tags` and remove this extra check here?

Reply:
Valid point, simplified in 89fe5deb16672678a2ad745d42d89401052ea276